clients/src/main/java/org/apache/kafka/common/protocol/types/Schema.java

```diff
@@ -64,14 +64,23 @@ public void write(ByteBuffer buffer, Object o) {
     }
 
     /**
-     * Read a struct from the buffer
+     * Read a struct from the buffer. Missing fields at the end of the buffer are replaced with
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6340 instead. Thanks!